cost/cloud_cost_anomaly_alerts/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.3
+
+- adding an input list for a rule based dimensions
+- adding an input list for a user dimensions
```

Comment:
Can we call standard dimension as "default dimension"

@lawrethao Can you please confirm ?